### PR TITLE
update recipe

### PR DESCRIPTION
**Pull request by isuruf**
@conda-forge-admin, please rerender

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, please rerender

**Comment by isuruf** [comment]
@ocefpaf, ready for review. Last commit should only affect osx.

**Comment by ocefpaf** [comment]
Wow! Passing!!! :tada:

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,8 +32,10 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - curl  # [win]
+    - python   3.6  # [osx]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Interesting that `osx` needs 3.6

Comment (isuruf) [reply recipe/meta.yaml]: I'm not entirely sure it's needed, but it's hard for me to debug in CI.

Comment (ocefpaf) [reply recipe/meta.yaml]: If it is working don't touch it :smile:

**Comment by ocefpaf** [comment]
@isuruf I noticed that you did not use multiple sources in the `meta.yaml` and kept the `qtwebkit` download in the `build.sh`.

We don't have to change it now b/c the build takes to long and this is working fine (planning to merge as soon as CircleCI finishes). I'm just wondering if there any reason for this so we do or do not try it again later.

File: .ci_support/linux_.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  sqlite:
+    max_pin: x
+  xz:
+    max_pin: x.x
+  zlib:
+    max_pin: x.x
+sqlite:
+- '3'
+xz:
+- '5.2'
```
Comment (ocefpaf) [inline .ci_support/linux_.yaml]: This is actually a bogus pinning b/c `xz` is only used to extract `qtwebkit`, but this is fine for now.

**Comment by isuruf** [comment]
> I'm just wondering if there any reason for this so we do or do not try it again later.

As I said indicated in my commit message, I'm just using a configuration that worked in your PR. I think we should just move to @mingwandroid's recipe for 5.6.3 soon, but this can be merged until we do.

**Comment by ocefpaf** [comment]
> As I said indicated in my commit message, I'm just using a configuration that worked in your PR.

Thought so, asked just b/c I know how hard it is to make changes in this recipe and still get a green build :smile: 

>  I think we should just move to @mingwandroid's recipe for 5.6.3 soon, but this can be merged until we do.

Yes! That is the plan :wink: 

Thanks to your input here I may be able to do that, unless you want to it :stuck_out_tongue_closed_eyes: 

(There is only one thing that I fear more than `gdal` and that is `qt`.)

**Comment by mingwandroid** [comment]
I'll add opencv and boost to that list esp on Windows

**Comment by ocefpaf** [comment]
> I'll add opencv and boost to that list esp on Windows

One day we need to get together for :beers: and drown our packing frustrations. I only wonder if there is enough beer in the world :thinking:

File: recipe/meta.yaml [hunk 3]
```diff
@@ -22,51 +22,48 @@ source:
...
 requirements:
   build:
-    - python  # [win]
-    - python 2.7*  # [unix]
+    - perl
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - curl  # [win]
+    - python   3.6  # [osx]
+    - python   2.7  # [linux]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Just curious, what is going on here? Why do we need two different version of Python for the two different OSes?

**Comment by mingwandroid** [comment]
To build webengine, it being a Google derived project, Python 2 is required. Not sure why it's not there for other OSes.

**Comment by ocefpaf** [comment]
> Do i need to do a Windows build for this?

It would be nice but I have no idea if that is going to be easy.

**Comment by gillins** [comment]
I've uploaded a package for `vc14` to the `gillins` channel (sha256: 6c352b4546fe32125e53214d778cd3f751b4f81ebe95991f6f48c9b28e89cdc7). I can't seem to build `vc9` any more due to the error below. Does any one know about this error with the latest `conda-build`? It used to work by looking for "Visual Studio for python 2.7" (which I do have installed) if VS9 (which I don't have) isn't available.

```
for /F "skip=2 tokens=2,*" %A in ('reg query "HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\VisualStudio\9.0" /v InstallDir') do SET "VSINSTALLDIR=%B"
ERROR: The system was unable to find the specified registry key or value.
```

**Comment by ocefpaf** [comment]
> I've uploaded a package for vc14 to the gillins channel (sha256: 6c352b4546fe32125e53214d778cd3f751b4f81ebe95991f6f48c9b28e89cdc7).

Thanks!
The package is in the channle: https://anaconda.org/conda-forge/qt/files

>  I can't seem to build vc9 any more due to the error below. Does any one know about this error with the latest conda-build? It used to work by looking for "Visual Studio for python 2.7" (which I do have installed) if VS9 (which I don't have) isn't available.

I guess you need to install "conda-forge-ci-setup=1" but I don't really know the Windows config stuff.

**Comment by isuruf** [comment]
@ocefpaf, the vc14 build has no pinnings.
@gillings, when you build do `conda build recipe -m /path/to/conda_build_config.yaml` where `conda_build_config.yaml` comes from the `conda-forge-pinning` package.

**Comment by isuruf** [comment]
Btw, how long does the windows build take?

**Comment by ocefpaf** [comment]
> @ocefpaf, the vc14 build has no pinnings.

Thanks for checking. I pulled the package. I may try to attempt to build it locally.

> Btw, how long does the windows build take?

Last time I checked it took more than 2 hours on my Windows machine :-(

**Comment by gillins** [comment]
I've uploaded a `qt-5.6.2-h2639256_8.tar.bz2` to the `gillins` channel for `vc14` (sha256: 153ba1e49fe20479be235751a0e7695c0187dbf53e04fa32ca4ce10a1a4b5ecd).

I've just seen that Visual Studio for Python 2.7 is no longer supported (https://github.com/conda/conda-build/issues/3039#issuecomment-407009373). I'm installing VS 2008 and will see if that fixes it.

**Comment by isuruf** [comment]
Looks good

**Comment by ocefpaf** [comment]
Thanks @gillins for building and @isuruf for reviewing!
Uploading it now :tada:

**Pull request merged**